### PR TITLE
Add `classList` + functions of `DOMTokenList`

### PR DESCRIPTION
**Pull request by sectore**
- All the following functions of `DOMTokenList` are added and supported by all modern
browsers (see https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList):
  * `add`
  * `remove`
  * `toggle`
  * `contains`
  * `item`

- incl. tests (depends on PR #105 )

domtokenlist-tests

### CODE REVIEW

File: test/Main.purs [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
 import Test.Unit.Assert (assert)
 import Test.Unit.Output.Simple (runTest)
-import Test.DOM.HTML.Window (domHtmlWindowTests)
-
 
-liftEff :: forall eff a. Eff (phantomjs :: PHANTOMJS | eff) a -> Aff (phantomjs :: PHANTOMJS | eff) a
-liftEff = EffClass.liftEff
 
+-- liftEff :: forall eff a. Eff (phantomjs :: PHANTOMJS | eff) a -> Aff (phantomjs :: PHANTOMJS | eff) a
+-- liftEff = EffClass.liftEff
```
Comment (garyb) [inline test/Main.purs]: Dead code

File: src/DOM/Node/DOMTokenList.purs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+foreign import contains :: forall eff. String -> DOMTokenList -> Eff (dom :: DOM | eff) Boolean
+
+foreign import toggle :: forall eff. String -> DOMTokenList -> Eff (dom :: DOM | eff) Boolean
+
+foreign import toggleForce :: forall eff. String -> Boolean -> DOMTokenList -> Eff (dom :: DOM | eff) Boolean
+
+foreign import _item :: forall eff. Int -> DOMTokenList -> Eff (dom :: DOM | eff) (Nullable String)
+
+item :: forall eff. Int -> DOMTokenList -> Eff (dom :: DOM | eff) (Maybe String)
+item index = map toMaybe <<< _item index
```
Comment (garyb) [inline src/DOM/Node/DOMTokenList.purs]: Can you rearrange the arguments in here (and accordingly in the FFI) so the `DOMTokenList` comes first in each case please?

**Review by garyb — CHANGES_REQUESTED** [review]
Thanks for working on this! Just a few minor things, and this PR needs a rebase since the test PR was merged.

**Comment by sectore** [comment]
@garyb No problem, latest commits ⬆️  come with these fixes (rebase + rearrange arguments).

**Comment by garyb** [comment]
Thank you!

**Review by garyb — APPROVED** [review]

**Pull request merged**